src/AlertBox/AlertBox.tsx

```diff
@@ -1,6 +1,6 @@
 import _ from "lodash";
 import classnames from "classnames";
-import React, {PureComponent, PropTypes} from "react";
+import * as React from "react";
```

Comment:
i think the others also need to be `import * as _ from "lodash";` and `import * as classnames from "classnames";`